src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/osgi/OsgiFramework.kt

```diff
@@ -145,6 +203,28 @@ class OsgiFramework(sync: InstanceSync) : InstanceService(sync) {
         }
     }
 
+    private fun getConfigList(): List<ConfigurationState> {
+        var configList: List<ConfigurationState> = listOf()
+        val regex = Regex("configData = (.*);")
+        sync.http.get(CONFIGURATION_PATH) { response ->
+            val mapper = ObjectMapper()
+            val html = asString(response)
+            val configJson = regex.find(html)?.groups?.get(1)?.value ?: "{}"
+            val json = mapper.readTree(configJson)
+            configList = mapper.readValue(json.get("pids").toString())
+        }
+        return configList
+    }
+
+    private fun getConfigPayload(configuration: OSGiConfiguration, properties: Map<String, Any>): Map<String, Any> {
```

Comment:
confusing method, looks like getter (`get` prefix is reserved for getters, do not use it when it is calculating sth)

Reply:
Name changed :)